packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -146,12 +147,14 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
   return (
     <BaseComposite {...props}>
       <CallAdapterProvider adapter={adapter}>
-        <MainScreen
-          callInvitationUrl={callInvitationUrl}
-          onFetchAvatarPersonaData={onFetchAvatarPersonaData}
-          onFetchParticipantMenuItems={onFetchParticipantMenuItems}
-          options={options}
-        />
+        <div className={props.options?.mobileView ? mainScreenContainerStyleMobile : mainScreenContainerStyleDesktop}>
```

Comment:
minor nit. can we determine the value of classname into a variable and just use the variable in this prop when we are returning our JSX.element?